hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1348,6 +1348,10 @@ private static void getSCMSignedCert(CertificateClient client,
     String hostname = omRpcAdd.getAddress().getHostName();
     String ip = omRpcAdd.getAddress().getHostAddress();
 
+    OMHANodeDetails haOMHANodeDetails = OMHANodeDetails.loadOMHAConfig(config);
+    String serviceName =
```

Comment:
In non-HA case serviceName will be null. Do we need to set in this case?

Reply:
Yes, good catch. if this is null, we should avoid setting this field in the certificate.

Reply:
Fixed